core/src/test/java/io/github/tcdl/msb/impl/RequesterImplTest.java

```diff
@@ -211,12 +218,12 @@ public void testRequestMessageWithTags() throws Exception {
         assertArrayEquals(new String[]{tag, dynamicTag}, requestMessage.getTags().toArray());
     }
 
-    private RequesterImpl<RestPayload> initRequesterForResponsesWithTimeout(int numberOfResponses) throws Exception {
+    private RequesterImpl<RestPayload> initRequesterForResponsesWith(Integer numberOfResponses, Integer respTimeout,  Integer ackTimeout , Callback<Void> endHandler) throws Exception {
 
         MessageTemplate messageTemplateMock = mock(MessageTemplate.class);
 
         RequestOptions requestOptionsMock = new RequestOptions.Builder().withMessageTemplate(messageTemplateMock).withWaitForResponses(numberOfResponses)
-                .withResponseTimeout(100).build();
+                .withResponseTimeout(respTimeout).withAckTimeout(ackTimeout).build();
```

Comment:
It would be better to split "with" and "build" method invokations into separate lines.